src/Go/boardAnalysis/goAI.ts

```diff
@@ -823,5 +823,5 @@ export function sleep(ms: number): Promise<void> {
 }
 
 export function showWorldDemon() {
-  return Player.augmentations.some((a) => a.name === AugmentationName.TheRedPill) && Player.sourceFileLvl(1);
+  return Player.augmentations.some((a) => a.name === AugmentationName.TheRedPill) && Player.sourceFileLvl(1) > 0;
```

Comment:
Is this fixing a bug? I don't see how this is any different, which has me worried.